Packs/CloudShare/Integrations/CloudShare/CloudShare.py

```diff
@@ -0,0 +1,1254 @@
+from typing import Any, Dict, List, Optional, Tuple, Union, cast
+
+import urllib3
+
+import cloudshare
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+''' IMPORTS '''
+
+
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+''' CONSTANTS '''
+
+
+''' CLIENT CLASS '''
+
+
+class Client():
+    def __init__(self, hostname, api_id=None, api_key=None, verify=True, proxy=False, ok_codes=tuple(), headers=None):
+        self.hostname = hostname
+        self.apiId = api_id
+        self.apiKey = api_key
+
+    def send_request(self, method, path, queryParams: dict = None, content: dict = None):
+        res = cloudshare.req(
+            hostname=self.hostname,
+            method=method,
+            path=path,
+            apiId=self.apiId,
+            apiKey=self.apiKey,
+            queryParams=queryParams,
+            content=content
+        )
+        return res
+
+
+''' HELPER FUNCTIONS '''
+
+
+def test_module_command(client, args):
+    res = client.send_request(
+        'GET',
+        'ping'
+    )
+    if res.status == 200:
+        if "result" in res.content and res.content['result'] == "Pong":
+            return_results('ok')
+        else:
+            return_error(res.content)
+    else:
+        return_error(res.content)
+
+
+def get_projects_command(client, args):
+    queryParams = {
+        "WhereUserIsProjectManager": True if args.get('WhereUserIsProjectManager', 'false') == 'true' else False,
+        "WhereUserIsProjectMember": True if args.get('WhereUserIsProjectMember', 'false') == 'true' else False,
+        "WhereUserCanCreateClass": True if args.get('WhereUserCanCreateClass', 'false') == 'true' else False
+    }
+    res = client.send_request(
+        'GET',
+        'projects',
+        queryParams=queryParams
+    )
+    if res.status == 200:
+        md = tableToMarkdown('CloudShare Projects:', res.content)
+        command_results = CommandResults(
+            outputs_prefix="CloudShare.Projects",
+            outputs_key_field='id',
+            outputs=res.content if res.content else None,
+            readable_output=md
+        )
+        return_results(command_results)
+    else:
+        return_error(f"Error getting projects - {res.content}")
+
+
+def get_project_command(client, args):
+    projectId = args.get('projectId')
+    res = client.send_request(
+        'GET',
+        f'projects/{projectId}'
+    )
+    if res.status == 200:
+        md = tableToMarkdown(f'CloudShare Project {projectId}:', res.content)
+        command_results = CommandResults(
+            outputs_prefix="CloudShare.Projects",
+            outputs_key_field='id',
+            outputs=res.content if res.content else None,
+            readable_output=md
+        )
```

Comment:
Please return the raw_result of the API alongside outputs under the `raw_response` for this command.

Please do the same for all subsequent functions.